.meshrc.yaml

```diff
@@ -144,6 +144,12 @@ sources:
             requestSample: {"selectedIds":[28114,28151],"workflow":"short-read-mngs","authenticityToken":"token"}
             responseSchema: ./json-schemas/validateUsersCanDeleteObjects.json
             responseTypeName: ValidateUserCanDeleteObjects
+          - type: Query
+            field: workflowRuns
+            path: /workflow_runs.json
+            method: POST
+            requestSchema: ./json-schemas/workflowRunsRequest.json
+            responseSchema: ./json-schemas/workflowRunsResponse.json
```

Comment:
I think @saq7 has been working on federating this endpoint in #92 as well.  we should probably see if we can get all the changes in one PR to avoid conflicts?

Reply:
From what we discussed offline, should we just merge this first? We can decide if we want to build off this and conditionally hit the other Rails endpoint, or just use a different federation endpoint for now?